### PR TITLE
[docs] Add short description of access  control.

### PR DESCRIPTION
**Pull request by maxlovic**
Based on the discussion with @justinjoy, adding a short description of the SRT Access Control Guidelines to specify the use case in short, before going into the details.

### CODE REVIEW

File: docs/AccessControl.md [hunk 1]
```diff
@@ -24,6 +24,17 @@ convention is to:
 - promote readability and consistency among free-form names
 - interpret some typical data in the key-value style
 
+In short,
+1. `SRTO_STREAMID` is designed for a caller (client) to be able to identify himself, and tell what he wants.
+2. `srt_listen_callback(...)` function is used by a listener (server) to check what a caller (client) has provided in `SRTO_STREAMID` **before** the connection is established.
+For example, the listener (server) can check if it knows the user amd set the corresponding passphrase for a connection to be accepted.
+3. Even if `srt_listen_callback(...)` accepts the connection, SRT will still have one more step to check the PASSPHRASE, and reject on mismatch.
```
Comment (justinjoy) [inline docs/AccessControl.md]: This is a cool part of preventing mix-up. I appreciate you helping me understand this feature.

Comment (justinjoy) [reply docs/AccessControl.md]: This is a part where SRT starts checking passphrase. 
https://github.com/Haivision/srt/blob/master/srtcore/api.cpp#L473

File: docs/AccessControl.md [hunk 2]
```diff
@@ -24,6 +24,17 @@ convention is to:
...
+If a correct passphrase is not provided by the client (caller), the request from caller will be rejected by SRT library (not application or programer).
+
+**Note!** `srt_listen_callback(...)` can't check the passphrase directly for security reasons. 
+The only way for the app to check the passprase is to tell the expected passprase in the callback, and let SRT handle mismatches.
```
Comment (ethouris) [inline docs/AccessControl.md]: I'd say it more precisely:

The only way for the app to make the passphrase checked is to set the passphrase on the socket by using the `SRTO_PASSPHRASE` option. This lets SRT handle mismatches.

Comment (maxlovic) [reply docs/AccessControl.md]: Applied.

File: docs/AccessControl.md [hunk 3]
```diff
@@ -24,6 +24,17 @@ convention is to:
 - promote readability and consistency among free-form names
 - interpret some typical data in the key-value style
 
+In short,
+1. `SRTO_STREAMID` is designed for a caller (client) to be able to identify himself, and tell what he wants.
```
Comment (stevomatthews) [inline docs/AccessControl.md]: `SRTO_STREAMID` is designed for a caller (client) to be able to identify itself, and state what it wants.

File: docs/AccessControl.md [hunk 4]
```diff
@@ -24,6 +24,17 @@ convention is to:
...
+2. `srt_listen_callback(...)` function is used by a listener (server) to check what a caller (client) has provided in `SRTO_STREAMID` **before** the connection is established.
+For example, the listener (server) can check if it knows the user and set the corresponding passphrase for a connection to be accepted.
+3. Even if `srt_listen_callback(...)` accepts the connection, SRT will still have one more step to check the PASSPHRASE, and reject on mismatch.
+If a correct passphrase is not provided by the client (caller), the request from caller will be rejected by SRT library (not application or programmer).
+
+**Note!** `srt_listen_callback(...)` can't check the passphrase directly for security reasons. 
+The only way for the app to make the passphrase checked is to set the passphrase on the socket by using the `SRTO_PASSPHRASE` option. This lets SRT to reject connection on mismatch.
```
Comment (stevomatthews) [inline docs/AccessControl.md]: The only way to make the app check the passphrase is to set the passphrase on the socket by using the

**Pull request merged**